### PR TITLE
Upcoming events front page

### PR DESCRIPTION
**Pull request by jburel**
Add upcoming events to front page.

### CODE REVIEW

File: index.html [hunk 1]
```diff
@@ -7,8 +7,12 @@
...
                 <div class="small-12 columns bg-image-text-container">
                     <h1 class="hero-main-header small-12">Next Workshop</h1>
-                    <p class="small-10 small-offset-1 medium-6 medium-offset-3">03-04 September in Basel (Switzerland)!</p>
-                    <p class="event-highlight"><a href="https://miap.eu/index.php?id=446" target="_blank" class="hero-link">See the schedule</a><p>
+                    <p class="small-10 small-offset-1 medium-6 medium-offset-3">03-04 September in Basel (Switzerland)!<a href="https://miap.eu/index.php?id=446" target="_blank" class="hero-link">See the schedule</a></p>
+                    <h2 class="hero-main-header small-12">Upcoming Workshops</h2>
+                    <p class="small-10 small-offset-1 medium-6 medium-offset-3">19-21 September in Sydney (Australia)</p>
+                    <p class="small-10 small-offset-1 medium-6 medium-offset-3">25 October in Kobe (Japan)</p>
+                    <p class="small-10 small-offset-1 medium-6 medium-offset-3">03 November in Okinawa (Japan)</p>
+                    <p class="small-10 small-offset-1 medium-6 medium-offset-3">06 December Cambridge (U.K.)</p>
```
Comment (mtbc) [inline index.html]: +"in"

File: index.html [hunk 2]
```diff
@@ -7,8 +7,14 @@
             <div class="row homepage text-center">
                 <div class="small-12 columns bg-image-text-container">
                     <h1 class="hero-main-header small-12">Next Workshop</h1>
-                    <p class="small-10 small-offset-1 medium-6 medium-offset-3">03-04 September in Basel (Switzerland)!</p>
-                    <p class="event-highlight"><a href="https://miap.eu/index.php?id=446" target="_blank" class="hero-link">See the schedule</a><p>
+                    <p class="small-10 small-offset-1 medium-6 medium-offset-3">03-04 September in Basel (Switzerland)!<a href="https://miap.eu/index.php?id=446" target="_blank" class="hero-link">See the schedule</a></p>
```
[the remark this answers is not in the document]
Comment (jburel) [reply index.html]: It was there before I will remove it

**Comment by jburel** [comment]
I kept the original "See the Schedule" to highlight the coming session

**Comment by pwalczysko** [comment]
Thinking that this is staged in https://snoopycrimecop.github.io/www.openmicroscopy.org/, the link in ``Kobe`` does not bring me to a valid page.

**Comment by pwalczysko** [comment]
screen shot 2018-08-24 at 20 13 47

Wouldn't it be better to adjust the lines to the left, instead of having it central ? If the dates were under each other, the calendar would be easier to grasp with one look.

**Comment by pwalczysko** [comment]
1 Bug, 1 RFE otherwise looks good.

**Comment by jburel** [comment]
I think the link was up the other day but it does not seem to be anymore
We will have to wait for Ken to give us the green light

**Comment by jburel** [comment]
@pwalczysko better?

**Comment by pwalczysko** [comment]
Looks better to me now, thank you.

File: events/index.html [hunk 3]
```diff
@@ -8,6 +8,16 @@
         <!-- begin Events -->
         <hr class="whitespace">
         <div id="community-posts" class="row">
+            <div class="small-12 medium-8 medium-offset-2 columns">
+                <h6><i class="fa fa-calendar"></i> November 03, 2018</h6>
+                <h2><a href="https://docs.google.com/spreadsheets/d/1KbX0lecANaqspUf-l2YYWEp3W4VaoBa1p06Hci7HSss/edit#gid=2059970125">OMERO Workshop, Okinawa</a></h2>
+                <p class="card-caption">A 2h workshop on Data Management with OMERO.</p>
+            </div>
+            <div class="small-12 medium-8 medium-offset-2 columns">
+                <h6><i class="fa fa-calendar"></i> October 19 & 21, 2018</h6>
```
Comment (sbesson) [inline events/index.html]: `s/October/September` ?

**Comment by sbesson** [comment]
I assume this page will now have to be updated every time we complete a training. Also unclear whether all the calendar of upcoming workshops should be maintained both on the front page banner and on the events page.

LGTM so that we can refer to these event pages and we can figure out how the maintenance works out.

**Comment by jburel** [comment]
We will have to update it but we need to promote better our various sessions
I am not sure yet about the events page either

**Pull request merged**